sklego/meta/zero_inflated_regressor.py

```diff
@@ -97,8 +102,14 @@ def fit(self, X, y, sample_weight=None):
                 f"`classifier` has to be a classifier. Received instance of {type(self.classifier)} instead."
             )
         if not is_regressor(self.regressor):
-            raise ValueError(f"`regressor` has to be a regressor. Received instance of {type(self.regressor)} instead.")
-
+            raise ValueError(
+                f"`regressor` has to be a regressor. Received instance of {type(self.regressor)} instead."
+            )
+        if not handle_zero in ['ignore', 'error']:
+            raise ValueError(
+                f" 'handle_zero' has to be one of 'ignore' or 'error'. Received {handle_zero} instead."
```

Comment:
Can we use backticks as well?

```suggestion
                f"`handle_zero` has to be one of {'ignore', 'error'}. Received '{handle_zero}' instead."
```